compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```
@@ -313,3 +313,4 @@ error.metadata.not.attached.to.a.object.member=metadata not attached to a object
 error.type.desc.after.rest.descriptor=type descriptor not allowed after rest descriptor
 error.configurable.var.implicitly.final=''final'' qualifier not allowed: configurable variables are implicitly final
 error.missing.attach.point.name=missing attach point name
+error.local.const.decl.not.allowed=local constant declaration not allowed
```

How about something like
```suggestion
error.local.const.decl.not.allowed=constant declarations are allowed only at module level
```

IMO, saying "local constant declaration" kinda implies the existence of a "local constant declaration" construct. WDYT?

---

+1 will update